src/components/List/List.scss

```diff
@@ -30,12 +31,13 @@
   border-radius: 25px;
   height: 6vh;
   width: 12vw;
+  // min-width
   cursor: pointer;
 }
 
 .list-container {
   background-color: #beab95;
-  height: 750px;
-  width: 750px;
+  height: calc(100vh - 125px);
+  width: calc(100vh - 125px);
```

Comment:
niice catch, we'll likely need to update these dimensions again to accommodate the drag-and-drop resizing we did